### PR TITLE
installed `annotate gem`

### PR DESCRIPTION
**Pull request by o-200**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.2.2"
 
+gem 'annotate', '~> 3.2'
```
Comment (XaoGao) [inline Gemfile:6]: @o-200 use double quotes

**Review by XaoGao — COMMENTED** [review]
@o-200 please fix

File: Gemfile (hunk 1 above)
Comment (o-200) [reply Gemfile]: > @o-200 use double quotes

fixed

File: Gemfile [hunk 2]
```diff
@@ -3,6 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.2.2"
 
+gem "annotate", '~> 3.2'
```
Comment (XaoGao) [inline Gemfile:6]: @o-200 anywhere double quotes
```
'~> 3.2'
```

**Review by XaoGao — COMMENTED** [review]
@o-200 did not fixed, be attention

File: Gemfile (hunk 2 above)
Comment (o-200) [reply Gemfile]: > @o-200 anywhere double quotes
> 
> ```
> '~> 3.2'
> ```

done

**Review by XaoGao — APPROVED** [review]
approve

**Pull request merged**